docs/Testing.rst

```diff
@@ -29,3 +29,29 @@ To test a PR, create a comment ``launch jenkins``.
 Supported machines:
 
 *   balfrin
+
+Jenkins test with uenv
+----------------------
+To test spack-c2sm with an uenv, add a stage
+
+.. code-block:: bash
+
+    stage('Create uenv') {
+        steps {
+            sh """
+            git clone -b fix/jenkins https://github.com/eth-cscs/uenv.git
```

Comment:
Can we expect this fix to be merged, is uenv versioned? would it be worthwile to stick to a version?

Reply:
You need to ask the developers of https://github.com/eth-cscs/uenv.git